pom.xml

```diff
@@ -252,6 +252,17 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
+    <repository>
+      <id>staged</id>
+      <name>staged-releases</name>
+      <url>https://repository.apache.org/content/repositories/staging/</url>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+    </repository>
```

Comment:
You should use the following, @wangyum .
```
    <repository>
      <id>hivetest</id>
      <url>https://repository.apache.org/content/repositories/orgapachehive-1096</url>
    </repository>
```

Reply:
Today, Alan uploaded here.

Reply:
Oh, it's uploaded there, too. Got it.
- https://repository.apache.org/content/repositories/staging/org/apache/hive/hive-common/2.3.6/